examples/docs/fixedAdrQvs.py

```diff
@@ -0,0 +1,13 @@
+import qupled.quantum as qpq
+
+# Define a QVSStls object to solve the QVSSTLS scheme
+qvsstls = qpq.QVSStls(0.1,
```

Comment:
rs = 0.1 might be good for testing but it doesn't make a good example. Try to have something like we have for VSStls, but maybe limit it to 32 matsubara frequencies, cutoff 5.0 and rs = 5.0 (this will probably not produce exact results but should be enough to show how the code works)

Reply:
My idea behind fixedAdrQvs.py was to use this as a way to compute the fixed component for any Theta. That's why I used rs=0.1 since this will just produce the 3 needed files and then run the qVS computation for a single state point.

Reply:
I think it's probably better to have everything in one single example. Maybe something along the line of what you have in the other example but not with two for loops